CONTRIBUTING.md

```diff
@@ -3,8 +3,8 @@
 The Feldera team welcomes contributions from the community. Before you start working with Feldera, please
 read our [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
 To acknowledge the DCO, sign your commits by adding `Signed-off-by: Your Name <your@email.com>` to the last
-line of each Git commit message.  Your signature certifies that you wrote the patch or have the right to pass
-it on as an open-source patch.  The e-mail address used to sign must match the e-mail address of the Git
+line of each Git commit message. Your signature certifies that you wrote the patch or have the right to pass
```

Comment:
why delete the extra spaces?
some editors, like emacs, actually use these to navigate sentences

Reply:
some editors, like mine clean this mess up. But I'll change my configuration so it doesn't disturb your editor :P